nedssoft/Politico
# Admin create political parties 163524062

#### What does this PR do?
Admin should be able to create a political party

#### Description of Task to be completed?

- Wrote a test to test for
      - Duplicate record while creating a new party
      - Ensure that the fields are not empty
      - Check for successful creating of a new party

- Created an endpoint for creating a new party
           
#### How should this be manually tested?
By running npm run test

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/163524062
